worker.py

```diff
@@ -1049,8 +1110,12 @@ def normalize_pokestop(raw):
     @staticmethod
     def check_captcha(responses):
         challenge_url = responses.get('CHECK_CHALLENGE', {}).get('challenge_url', ' ')
-        if challenge_url != ' ':
-            raise CaptchaException
+        success = responses.get('VERIFY_CHALLENGE', {}).get('success')
+        if challenge_url != ' ' and not success:
```

Comment:
Is a URL still sent back on a successful `VERIFY_CHALLENGE` call? If not, the `success` checking is unnecessary.